Script/char.dat

```diff
@@ -545,10 +553,13 @@ guard
     ClothColor = rgb16(130, 80, 30);
     FriendlyReplies =
     {
-      4,
+      7,
       "@Dd says sadly: \"Back then I used to love bananas. One day the master guard slipped on a peel I'd dropped. Guess where I got transferred?\"",
       "\"Lions killed a couple of banana growers again. Hope they're now satisfied and don't attack us.\"",
       "\"This place's as hot as the elemental plane of fire! I want back home.\"",
+      "\"What lousy pay we're getting here!\"",
+      "\"The food's not fit for orcs!\"",
+      "\"My feet hurt, I've been on them all day!\"",
       "\"New Attnam, shit, I'm still only in New Attnam. Every time I think I'm gonna wake up back in the dungeon.\"";
```

Comment:
This is the only instance of `shit` in game text; probably it should be removed.